src/app/[locale]/(marketing)/about/page.tsx

```diff
@@ -1,4 +1,10 @@
+import tcm_bigbag from '@/public/assets/images/tcm-bigbag.jpg';
```

Comment:
https://www.youtube.com/watch?app=desktop&v=gEMAZSO85KY&t=0s

This might be helpful